src/main/resources/static/js/main.js

```diff
@@ -113,6 +113,7 @@ jQuery(document).ready(function() {
     }
 
     function viewModel(data) {
+    console.log(data);
```

Comment:
I guess this is just for debugging for the moment.

Reply:
removed